joladnijo/settings.py

```diff
@@ -0,0 +1,127 @@
+"""
+Django settings for joladnijo project.
+
+Generated by 'django-admin startproject' using Django 3.2.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/3.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/3.2/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/3.2/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-gu-7-g5=o3g2=b$!@j^#2c)y*^#+$u9&a7klfrla2dr31@h+as'
```

Comment:
Ez maradhat most itt?

Reply:
Jogos a kérdés. Mivel most a cél, hogy el tudjunk kezdeni párhuzamosan dolgozni a projekten, nem foglalkoznék vele. Esetleg egy ticket-et megér, hogy ne felejtsünk el foglalkozni vele.

Reply:
Ha ez a SECRET_KEY az eles kornyezetben mas erteket kap, akkor maradhat. Viszont akkor meg erdemes lenne valami trivialisabb erteket adni. Pl. `django-insecure-key-for-test`

Reply:
Módosítottam